bucket/activemq-artemis.json

```
@@ -0,0 +1,25 @@
+{
+    "version": "2.24.0",
+    "description": "Flexible & Powerful Open Source Multi-Protocol Messaging",
+    "homepage": "https://activemq.apache.org/components/artemis/",
+    "license": "Apache-2.0",
+    "suggest": {
+        "JDK": "java/openjdk"
+    },
+    "url": "https://archive.apache.org/dist/activemq/activemq-artemis/2.24.0/apache-artemis-2.24.0-bin.zip",
+    "hash": "sha512:97e92d8e013dd44489718b5dd389064e1c9ca7944cffd10cc6f14c9b4aaefe722ae16aa2fd825e0b625750dc18e364e0f166df59695a64d0554157055745b50b",
+    "extract_dir": "apache-artemis-2.24.0",
+    "bin": "bin\\artemis.cmd",
+    "checkver": {
+        "url": "https://activemq.apache.org/components/artemis/download/",
+        "regex": "ActiveMQ Artemis ([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://archive.apache.org/dist/activemq/activemq-artemis/$version/apache-artemis-$version-bin.zip",
+        "extract_dir": "apache-artemis-$version",
+        "hash": {
+            "url": "$url.sha512"
+        }
+    },
+    "notes": "Create a broker instance by using artemis create. The broker cannot be persisted!"
```

```suggestion
    "notes": "Create a broker instance by running 'artemis create (broker path)'"
```

> I'd suggest leaving it up to the user to create the broker in a safe location.

I agree with you,
but I think we don't have to mention `The broker cannot be persisted`.

If the user creates a new broker by running `artemis create broker1`. The folder `broker1` will be created under **current working directory**, so it is expected that Scoop's persist machanism will not handle it.

---

I completely agree, I've added your suggested changes!